website/profile/views.py

```diff
@@ -280,6 +283,8 @@ def user_profile(auth, **kwargs):
 def user_account(auth, **kwargs):
     user = auth.user
     user_addons = addon_utils.get_addons_by_config_type('user', user)
+    if 'password_reset' in request.query_string:
```

Comment:
Still not sure the best way to do this. Resetting your password logs you out and then logs you back in.  We still need to push a status message that your password was changed.   If your password was changed successfully, I am adding a query_params `?password_reset=True`. If that query param is present on this route, then the "password updated successfully" message will be visible.

Reply:
Nit: `'password_reset' in request.args` is preferred because `request.args` is a dict of the query params whereas `request.query_string` is a bytes string.